mrbgems/mruby-test/mrbgem.rake

```diff
@@ -7,10 +7,7 @@ MRuby::Gem::Specification.new('mruby-test') do |spec|
   spec.add_dependency('mruby-compiler', :core => 'mruby-compiler')
 
   spec.test_rbfiles = Dir.glob("#{MRUBY_ROOT}/test/t/*.rb")
-  if build.cc.defines.flatten.include?("MRB_WITHOUT_FLOAT")
-    spec.test_rbfiles.delete("#{MRUBY_ROOT}/test/t/float.rb")
-  end
-
+  spec.test_rbfiles.delete("#{MRUBY_ROOT}/test/t/float.rb")
```

Comment:
Skipping `test/t/float.rb` in any case is not desirable, I think.  Am I missing something?

Reply:
Sorry, I made a mistake. I will recreate the pull request.